### PR TITLE
Add first robotic tag

### PR DESCRIPTION
**Pull request by osanseviero**
For reference: https://twitter.com/theCamusean/status/1575177398852943872 

For projects such as
* https://cliport.github.io/
* https://github.com/NVlabs/6dof-graspnet

Right now the modality is `rl`, but we will rename it to a name that better encompasses both robotics and RL.

Screenshot from 2022-10-03 23-30-50
Screenshot from 2022-10-03 23-30-36

### CODE REVIEW

**Review by natolambert — APPROVED** [review]
Looks good for an initial solution 🚀

**Comment by osanseviero** [comment]
This needs some work (e.g. the icon)  from my side, I'll get to it soon!

**Review by mishig25 — APPROVED** [review]

**Comment by julien-c** [comment]
can we see a screenshot of the icon? =)

File: js/src/lib/components/Icons/IconRobotics.svelte [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	xmlns="http://www.w3.org/2000/svg"
+	xmlns:xlink="http://www.w3.org/1999/xlink"
+	aria-hidden="true"
+	fill="currentColor"
+	focusable="false"
+	role="img"
+	width="1em"
+	height="1em"
+	preserveAspectRatio="xMidYMid meet"
+	viewBox="0 0 32 32"
```
Comment (mishig25) [inline js/src/lib/components/Icons/IconRobotics.svelte:16]:
```suggestion
	viewBox="0 0 24 24"
```

Fixes the slight top left tilt in this msg,
Now it fills the entire "square"
image

**Comment by osanseviero** [comment]
I added 2 screenshots with a built instance of the Hub. I also fixed the documentation on this as it was outdated and added `robotics` in a couple of other places

File: js/src/lib/components/Icons/IconRobotics.svelte (hunk 1 above)
Comment (gary149) [reply js/src/lib/components/Icons/IconRobotics.svelte]: can we see it in context?

Comment (osanseviero) [reply js/src/lib/components/Icons/IconRobotics.svelte]: I added screenshots at the top

**Comment by julien-c** [comment]
i think it's too small compared to the other ones, need to crop it a little bit (or change to another icon, but this one is ok IMO) @gary149

**Comment by osanseviero** [comment]
Some help in that would be greatly appreciated :)

**Comment by gary149** [comment]
> Some help in that would be greatly appreciated :)

Should be better now.

**Comment by osanseviero** [comment]
Thanks!

**Review by julien-c — APPROVED** [review]

**Comment by julien-c** [comment]
Updated screenshot in context: 
image

**Comment by osanseviero** [comment]
Let's ship it! :robot:

**Pull request merged**